posix.mak

```diff
@@ -36,11 +36,11 @@ STABLE_RDMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODE
 
 # exclude lists
 MOD_EXCLUDES_PRERELEASE=$(addprefix --ex=, gc. rt. core.internal. core.stdc.config core.sys.	\
-	std.c. std.algorithm.internal std.internal. std.regex.internal. std.typelist		\
+	std.c. std.algorithm.internal std.internal. std.regex.internal. 			\
 	std.windows.iunknown std.windows.registry etc.linux.memoryerror std.stream std.cstream	\
```

Comment:
Not at all off-topic. It's excluded because it's a Windows-only module, and unlike the other Windows-only modules not excluded here, it doesn't have a `version(StdDdoc)` block which only has the documentation.